cctools/libmacho/Makefile

```diff
@@ -39,6 +39,10 @@ CFLAGS = $(OFLAG) -g -Wall $(LEGACY) \
 	 -I$(SRCROOT) -I$(SRCROOT)/../include -I$(OFILE_DIR) $(SDK)
 PG = -pg
 
+ifeq "$(RC_MACOS)" "YES"
+  DYNAMICLIB_FLAGS = -Wl,-iosmac_version_min,12.0
```

Comment:
The variable being checked says MACOS, but the flag being used is an iOS one? Is that correct?